server/command.go

```diff
@@ -246,3 +247,25 @@ func (p *Plugin) ExecuteCommand(_ *plugin.Context, args *model.CommandArgs) (*mo
 	p.postCommandResponse(args, "Unknown action %v", action)
 	return &model.CommandResponse{}, nil
 }
+
+func getAutocompleteData() *model.AutocompleteData {
+	welcomebot := model.NewAutocompleteData("welcomebot", "[command]",
+		"Available commands: preview, help, list, set_channel_welcome, get_channel_welcome, delete_channel_welcome")
+
+	preview := model.NewAutocompleteData("preview", "[team-name]", "Preview the welcome message for the given team name")
```

Comment:
`team-name` should be a text argument similar to https://github.com/mattermost/mattermost-plugin-github/blob/6957b7f8e97a96030982b1c527713e486e41afbd/server/plugin/command.go#L541

Reply:
Committed at https://github.com/mattermost/mattermost-plugin-welcomebot/pull/69/commits/ffeadbc340f17b1c44c62fa9f7c586b85aadb859